### PR TITLE
fix empty caplog

### PR DESCRIPTION
**Pull request by CunliangGeng**
Fix failed unit test which is caused by not setting logging in unit tests.

### CODE REVIEW

**Comment by CunliangGeng** [comment]
* **#260** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/260?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#259** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/259?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#258** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/258?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#257** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/257?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#256** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/256?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#255** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/255?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#254** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/254?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#253** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/253?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#252** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/252?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#250** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/250?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#249** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/249?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#248** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/248?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#247** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/247?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#246** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/246?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#245** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/245?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @CunliangGeng and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: tests/unit/schemas/test_user_strains_schema.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # Test schema aginast invalid data
 data_no_strain_ids = {"version": "1.0"}
 data_empty_strain_ids = {"strain_ids": [], "version": "1.0"}
-data_invalid_strain_ids = {"strain_ids": [1, 2, 3], "version": "1.0"}
+data_invalid_strain_ids = {"strain_ids": [1, ], "version": "1.0"}
```
Comment (gcroci2) [inline tests/unit/schemas/test_user_strains_schema.py:10]: Why 2 and 3 have been removed? I guess because testing on 1 is enough, but to be sure

**Review by gcroci2 — APPROVED** [review]

File: tests/unit/schemas/test_user_strains_schema.py (hunk 1 above)
Comment (CunliangGeng) [reply tests/unit/schemas/test_user_strains_schema.py:10]: Yes, only `1` is enough. And the new version of jsonschema changes the order of a list, making the test with multiple items challenging.

**Comment by CunliangGeng** [comment]
### Merge activity

* **Jun 6, 3:12 AM EDT**: @CunliangGeng started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/NPLinker/nplinker/245).
* **Jun 6, 3:13 AM EDT**: @CunliangGeng merged this pull request with [Graphite](https://app.graphite.dev/github/pr/NPLinker/nplinker/245).

**Pull request merged**